### ISSUE TITLE
What is the Const node in batch_normalization?

### ISSUE THREAD

**Issue by shinel94** [tensorflow/tensorflow#33450]
use **tensorflow.layer.batch_normalization** then define the batch_normalization/Const node,
<br>
## what is this const node and how act this node in batch normal operation?

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
